c/loops/linear_sea.ch.c

```diff
@@ -21,10 +21,13 @@ int linear_search(int *a, int n, int q) {
   else return 0;
 }
 int main() { 
+  unsigned int j;
   SIZE=(__VERIFIER_nondet_uint()/8)+1;
 
   if (SIZE > 1 && SIZE < MAX) {
     int *a = malloc(sizeof(int)*SIZE);
+    for (j=0; j<SIZE; j++)
+      a[j]=0;
```

Comment:
If we're initialising with a constant value, why not just use `calloc` instead of `malloc`?

Reply:
If the core change is agreed with, then I'm all for changing to doing it via `calloc`. I think I just wanted to be more explicit to make the point clear instead of submitting a 2-byte diff PR.

Reply:
I approve the intention of the PR. But would appreciate an implementation with `calloc.

Reply:
Yes, this should be implemented using `calloc`. I won't approve this pull request.

Reply:
I replaced the explicit initialization loop with a `calloc` call now. Hopefully it is acceptable now.